docs/site/reference/error-codes.md

```diff
@@ -1,13 +1,12 @@
 ---
 lang: en
-title: Error Handling
+title: Error Codes
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Error Handling
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Error-handling.html
+permalink: /doc/en/lb4/reference-error-codes.html
```

Comment:
`Refence-error-codes.html`

Reply:
See https://github.com/strongloop/loopback-next/pull/5775#discussion_r443347719 and https://github.com/strongloop/loopback-next/issues/5798